src/components/VolunteerDashbord/IndividualVolunteers/styles.scss

```diff
@@ -0,0 +1,143 @@
+.volunteer-class-page-container {
+  padding: 0 10%;
+  margin: 40px auto 0;
+}
+
+.volunteer-table-container {
+  padding: 0 10%;
+  margin: 40px auto 0;
+}
+
+.volunteer-page-heading {
+  display: flex;
+  font-size: 18px;
+  line-height: 28px;
+  font-weight: 400;
+}
+
+.home {
+  color: #48a145;
+  text-decoration: none;
+}
+
+.volunteer-details {
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: 400;
+}
+
+.volunteer-name {
+  font-weight: 700;
+}
+
+.volunteer-search-box {
+  border: 0;
+  font-size: 16px;
+  outline: 0;
+  background: transparent;
+  border-bottom: 1px solid #2e2e2e;
+  height: 5px;
+  margin-top: 13px;
+  width: 1280px;
+  background-image: url("../../../asset/search.svg");
+  background-repeat: no-repeat;
+  padding: 12px 20px 12px 40px;
+}
+
+.volunteer-class-table {
+  margin-top: 30px;
+  margin-bottom: 30px;
+  width: 100%;
+  table-layout: fixed;
+  border-collapse: collapse;
+}
+
+.volunteer-class-table td,
+th {
+  padding: 14px;
+  border: none;
+  font-family: Rubik;
+  font-style: normal;
+  font-size: 16px;
+}
+
+.volunteer-class-table thead tr th {
+  letter-spacing: 0.35;
+  line-height: 20px;
+  color: #2e2e2e;
+  // opacity: 1;
+  padding: 12px;
+  font-size: 16px;
+  font-style: normal;
+  // font-family: Nunito Sans;
+  text-align: start;
+  vertical-align: top;
+  // border: 1px solid #eb0f0f;
+  border-bottom: 2px solid #e6e3e3;
+  // border: none;
+}
+
+.volunteer-class-table tbody {
+  border-bottom: 2px solid #e6e3e3;
+}
+
+.volunteer-class-table tr:hover {
+  background-color: #e9f5e9;
+  cursor: pointer;
+  box-shadow: 0px 0px 0px 0px;
+}
+
+.sort-class {
+  background: transparent;
+  border: none;
+}
+
+@media only screen and (max-width: 760px),
+  (min-device-width: 768px) and (max-device-width: 1024px) {
```

Comment:
Okay, if we can't use it now, what can we use instead?

Reply:
@Poonam-Singh-Bagh Why are we using `device-width` ? For responsiveness in device orientation. From what I understand, `device-width` takes your device screen size into account. If you want to target the viewport you can simply use `max-width` and `min-width` (or `max-height` and `min-height`)

Reply:
I don't know Komal has used device-width in Partners' Dashboard and elsewhere we are using `@media screen and (max-width: 768px)` . I'm replacing `device-width` with `@media screen and (max-width: 768px)`

Reply:
@Poonam-Singh-Bagh As @pysaquib mentioned, `max-width`, `min-width`, `max-height`, and `min-height` are generally what we care about when making a responsive site since they are sensitive to changes in browser dimensions, allowing for changes to the layout to be applied on the fly during resizing, while `device-width` and `device-height` are not.  If the point of using `device-width` was to target say iPads, then a replacement could be to use a combination of other media features that might distinguish them (e.g., [hover](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/hover) of `none`, [resolution](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/resolution) of `192dpi`) but this almost certainly won't be 100% accurate as these features could be shared by non-iPads and are subject to changes.  However, this is also why `device-width` wasn't too useful for distinguishing devices, as dimensions of screens are not necessarily specific to say iPads and newer versions of these devices (e.g., iPad Pros) can wind up having different dimensions.